### DISCUSSION TITLE
How are fusions defined by SQANTI?

### DISCUSSION THREAD

**Discussion by tky199996** [ConesaLab/SQANTI3#286] [Q&A] [answered]
Recently I was looking for fusion genes, I found fusion genes using pbfusion, but I found that there seems to be a fusion structure type in the SQANTI3, I looked at the corresponding gene annotations, and found that the results of the two software are completely different, I thought about a gene that SQANTI3 the fusion transcript that this software is looking for is different from the fusion gene that is directly searching?

**Comment by carolinamonzo** [comment, marked as answer]
Hi @tky199996, 
SQANTI classifies as fusions those reads that map to two different genes, and that information is noted in the GTF file, so the annotated_gene column of that read has two genes. PBfusion is specific for this task, it not only looks for transcripts with two different annotations, but also strand switches, and novel exons are clustered/chained by breakpoints. SQANTI is not a gene fusion discovery tool, it's a quality control tool that uses fusions as a measure of quality of sequencing, so it's not the best at this, it's good enough to generate the category information and learn if the sequencing was sucessful. There are other tools (longGF, JAFFAL, FusionSeeker, FUGAREC, CTAT-LR-fusion etc) specific for this task.
